templates/managed/index.html

```diff
@@ -134,455 +155,421 @@ <h1>Managed IT Services</h1>
-              alt="Canonical Kubernetes",
-              width="271",
-              height="42",
-              hi_def=True,
-              loading="lazy",
-              ) | safe
-            }}
+        </li>
+        <li class="p-stepped-list__item">
+          <hr class="p-rule">
+          <h3 class="p-heading--5 p-stepped-list__title">
+            Management
+          </h3>
+          <div class="p-stepped-list__content">
+            Our team manages all your ‘day 2’ operations, from observability to upgrades. We handle all issues, and
+            consistently re-enforce the security of your environment.
           </div>
-          <hr />
-          <p>Deploy Kubernetes on bare metal, public cloud or on top of your OpenStack, and maintain your focus on developing your cloud native applications.</p>
-          <p><a class="p-button--positive" href="/kubernetes/managed">Managed Kubernetes</a></p>
-        </div>
-
-        <div class="col-4 p-card">
-          <div class="u-sv1">
-            {{
-              image (
-              url="https://assets.ubuntu.com/v1/dfd3bd7a-canonical-ceph.svg",
-              alt="Canonical Ceph",
-              width="199",
-              height="42",
-              hi_def=True,
-              loading="lazy",
-              ) | safe
-            }}
+        </li>
+        <li class="p-stepped-list__item">
+          <hr class="p-rule">
+          <h3 class="p-heading--5 p-stepped-list__title">
+            Optional handover
+          </h3>
+          <div class="p-stepped-list__content">
+            We’re lock-in free, so if after a while you want to start managing your environments by yourself, we can
+            teach you how and give you remote support for high-severity cases.
           </div>
-          <hr />
-          <p>As easy as the public cloud, just more economical. Fully managed open-source software-defined storage, with a proven reference architecture, that can be customised for your needs.</p>
-          <p><a class="p-button--positive" href="/ceph/managed">Managed Ceph</a></p>
-        </div>
+        </li>
+      </ol>
+    </div>
+  </div>
+</section>
 
-        <div class="col-4 p-card">
-          <div class="u-sv1">
-            {{
-              image (
-              url="https://assets.ubuntu.com/v1/ea7adcf4-Canonical+Landscape+dark+text.svg",
-              alt="Canonical Landscape",
-              width="268",
-              height="42",
-              hi_def=True,
-              loading="lazy",
-              ) | safe
-            }}
-          </div>
-          <hr />
-          <p>Landscape provides a dashboard and API for a wide range of administrative functions that encompass inventory, automation, hardening, compliance, reporting, and software distribution</p>
-          <p><a class="p-button--positive" href="/landscape/managed">Managed Landscape</a></p>
+<!-- Keep hidden until isometric illustrations arrive -->
```

Comment:
I think it's better to remove commented out code as it causes mess within the code base. We can use source control to recover the removed mark up.